Skyscanner/backpack-react-native
# [No JIRA] Upgrade to Storybook 5

I've had to change dependencies, have commented inline to explain each.

## How to test

1. Check out this branch.
2. Reinstall deps with `npm run clean && npm i`.
3. `npm start`
4. In a new tab `npm run android && npm run ios`
5. Confirm everything's 👌
